backend/.pre-commit-config.yaml

```diff
@@ -23,5 +23,6 @@ repos:
     rev: v0.931
     hooks:
       - id: mypy
+        files: ^backend/
```

Comment:
I guess we add `loader` and `pipeline` later?

Reply:
yeah, we'll have to, they have a good chunk of errors themselves